apps/create-vesting/src/app/page.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
+
+import {
+  type CoWHookDappActions,
+  type HookDappContext,
+  initCoWHookDapp,
+} from "@cowprotocol/hook-dapp-lib";
+
+import {
+  Input,
+  PeriodWithScaleInput,
+  ButtonPrimary,
+  ContentWrapper,
+  TokenAmountInput,
+  Wrapper,
+} from "@bleu/cow-hooks-ui";
+
+import { useEffect, useState } from "react";
+
+import { Form } from "@bleu/ui";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+
+import { createVestingSchema, periodScaleOptions } from "#/utils/schema";
+
+//will be used
+const scaleToSecondsMapping = {
+  Day: 24 * 60 * 60,
+  Week: 7 * 24 * 60 * 60,
+  Month: 30 * 24 * 60 * 60,
+};
+
+export default function Page() {
+  const [actions, setActions] = useState<CoWHookDappActions | null>(null);
+  const [context, setContext] = useState<HookDappContext | null>(null);
+
+  const isDarkMode = context?.isDarkMode;
+  // will be used
+  const { account, chainId } = context || {};
+
+  const form = useForm<typeof createVestingSchema._type>({
+    resolver: zodResolver(createVestingSchema),
+    defaultValues: {
+      period: 1,
+      periodScale: "Day",
+    },
+  });
+
+  const { handleSubmit } = form;
+
+  useEffect(() => {
+    const { actions } = initCoWHookDapp({ onContext: setContext });
+    setActions(actions);
+  }, []);
+
+  useEffect(() => {
+    const newTheme = isDarkMode ? "dark" : "light";
+    document.documentElement.setAttribute("data-theme", newTheme);
+  }, [isDarkMode]);
+
+  const addHook = () => {
+    if (!actions) return;
+
+    const hook = {
+      target: "",
+      callData: "",
+      gasLimit: "",
+    };
+    // will be used
+    // if (context?.hookToEdit) {
+    //   actions.editHook({ hook, uuid: context.hookToEdit.uuid });
+    // } else {
+    //   actions.addHook({ hook });
+    // }
+  };
+
+  function onSubmit(data: typeof createVestingSchema._type) {
+    if (!context) return;
+    console.log("data", data);
+    addHook();
+  }
+
+  return (
+    <>
+      {context && (
+        <Form {...form} className="contents">
+          <Wrapper>
+            <ContentWrapper>
+              <Input
+                name="recipient"
+                label="Recipient"
+                placeholder="0xabc..."
+                autoComplete="off"
+                className="w-full h-12 mt-0 p-2.5 rounded-xl outline-none text-color-text-paper border-2 border-color-border bg-color-paper-darker placeholder:opacity-100"
```

Comment:
There is a lot of stylling already with bleu/ui component. Are the styling work? 🤔